northd/northd.c

```diff
@@ -1834,6 +1834,12 @@ localnet_can_learn_mac(const struct nbrec_logical_switch_port *nbsp)
     return smap_get_bool(&nbsp->options, "localnet_learn_fdb", false);
 }
 
+static bool
+enable_conntrack(const struct nbrec_logical_switch_port *nbsp)
```

Comment:
We probably don't need this additional function.  If we populate a field in the port we could do call `smap_get_bool()` directly in the function doing that (please see my following comment).

Reply:
ok!